manifests/stable/service-binding-operator.v0.0.11.clusterserviceversion.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   annotations:
     alm-examples: '[{"apiVersion":"apps.openshift.io/v1alpha1","kind":"ServiceBindingRequest","metadata":{"name":"example-servicebindingrequest"},"spec":{"applicationSelector":{"matchLabels":{"connects-to":"postgres","environment":"production"},"resourceKind":"Deployment"},"backingSelector":{"resourceName":"database.example.org","resourceVersion":"v1alpha1"}}}]'
     capabilities: Basic Install
-    containerImage: quay.io/redhat-developer/app-binding-operator:v0.0.10
+    containerImage: quay.io/redhat-developer/app-binding-operator:v0.0.11
```

Comment:
Should be good, since I've created a tag in Quay.

Reply:
Yes, I saw that in quay, so that's why I chose that version as `stable` :).